app/views/spree/shipment_mailer/shipped_email.html.haml

```diff
@@ -0,0 +1,19 @@
+= Spree.t('shipment_mailer.shipped_email.dear_customer')
+
+= Spree.t('shipment_mailer.shipped_email.instructions')
+
+= "============================================================"
+= Spree.t('shipment_mailer.shipped_email.shipment_summary')
+= "============================================================"
+- @shipment.manifest.each do |item|
+  = item.variant.sku
+  = item.variant.product.name
+  = item.variant.options_text
+= "============================================================"
+
+- if @shipment.tracking
+  = Spree.t('shipment_mailer.shipped_email.track_information', tracking: @shipment.tracking)
+- if @shipment.tracking_url
+  = Spree.t('shipment_mailer.shipped_email.track_link', url: @shipment.tracking_url)
+
+= Spree.t('shipment_mailer.shipped_email.thanks')
```

Comment:
Are you sure switching from .text to .HTML won't break things? I never investigated this but I think .text is used as a fallback text-version of the email for specific old mail servers/clients. Again, no idea about how it works.

Reply:
Spree emails are all text, and ofn's are html. So I think it's fine, from now on these will also be sent in html. Testing will validate that as well.

I think you are referring to something we _could_ do which is to send html and text emails and the email client would use text as a fall back option. This is also good for deliverability but it's something we are not doing and is out of scope for this PR. Makes sense?

Reply:
Understood, I thought those text ones were just a fallback and not Spree's only templates :+1: Then, this means that those only-text emails saw Eugeni some time ago were a bug or they were missing an OFN template :see_no_evil:

Reply:
The emails seen by Eugeni are this shipment email. It was text in spree and after this PR it's html in OFN. But still without the normal OFN styling. That will need to be addressed in another PR.